bin/phpstan-baseline-trend.php

```diff
@@ -32,5 +34,27 @@
     exit(254);
 }
 
-$exitCode = $app->start($argv[1], $argv[2]);
+$exitCode = $app->start($argv[1], $argv[2], extractOutputFormat($argv));
 exit($exitCode);
+
+/**
+ * @param list<string> $args
+ * @return \staabm\PHPStanBaselineAnalysis\TrendApplication::OUTPUT_FORMAT_*
+ */
+function extractOutputFormat(array $args): string
+{
+    foreach($args as $arg) {
+        if (false === strpos($arg, '--format=')) {
```

Comment:
the regular baseline command uses just `—json`, see https://github.com/staabm/phpstan-baseline-analysis/blob/7a5201ef4b4a4907a47df6ce068a2a0c48c849a2/README.md?plain=1#L61

Could you add a `—format` option in the commands currently supporting only `—json`?